framework/src/util/AnyMap.cpp

```diff
@@ -146,7 +146,7 @@ Any AtCompoundKey(const std::vector<Any>& v,
         ref_any_cast<std::vector<Any>>(h), tail, std::move(defaultval));
     }
   }
-  return defaultval;
+  return std::move(defaultval);
```

Comment:
what's the purpose of these changes? They don't seem to have any relation to the benchmark tests.

Reply:
Without that change I get the following warning/error:

      local variable 'defaultVal' will be copied despite being returned by name
      [-Werror,-Wreturn-std-move]